annex/include/temperature_data.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TEMPERATURE_DATA_H
```

Comment:
This file is not used anywhere. Remove it.

Reply:
Done